README.md

```diff
@@ -22,12 +33,134 @@ It supports TensorFlow, PyTorch, MXNet, and XGBoost on Python 3.6+.
 - Distributed training support
 - TensorBoard support
 
+See [How it works](#how-it-works) for more details.
+
+## Support
```

Comment:
Should this be so high? Seems like something you'd put last on the readme. Also shouldn't it be "Release Notes" instead of Support?